L1_Syntax, Variables, and Numbers.ipynb

```diff
@@ -260,7 +260,7 @@
    "source": [
     "It always gives us a `float`. \n",
     "\n",
-    "The `//` operator gives us a result that's rounded down to the next integer."
+    "The `//` operator gives us a result that's rounded down to the previous integer."
```

Comment:
That was a typo, it should be like below. You can also add in parenthesis before the "." (floor division)
The // operator gives us a result that's rounded down to the nearest integer.

Reply:
yes, that is correct. will change it to:
"The // operator gives us a result that's rounded down to the nearest integer."